model/Software/Classes/PackageVerification.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageVerification
```

Comment:
@maxhbr - If you could propose some specific text that implements your proposal - I would like to close this and the related issue.

Reply:
I would suggest to drop the file `PackageHash.md`. And to make `PackageGitoid` and `PackageVerificationCode` a subclass of `Hash`

Reply:
@maxhbr - I'm about to head out on travel without email - do you want to open a new PR with your proposal and I'll close this one?

Reply:
Just found the issue that: if `PackageVerificationCode` extends `Hash` it then needs to set the `hashAlgorithm`. I think it would be correct and OK to set it on `sha1` but that might also be confusing. Not sure what to do